diff --git a/node_modules/@react-native-firebase/database/lib/DatabaseDataSnapshot.js b/node_modules/@react-native-firebase/database/lib/DatabaseDataSnapshot.js
index a555d04..307de7c 100644
--- a/node_modules/@react-native-firebase/database/lib/DatabaseDataSnapshot.js
+++ b/node_modules/@react-native-firebase/database/lib/DatabaseDataSnapshot.js
@@ -21,8 +21,8 @@ import {
   isObject,
   isString,
   isNumber,
-} from '@react-native-firebase/app/lib/common';
-import { deepGet } from '@react-native-firebase/app/lib/common/deeps';
+} from '@react-native-firebase/app/lib/common/index.js';
+import { deepGet } from '@react-native-firebase/app/lib/common/deeps.js';
 
 export default class DatabaseDataSnapshot {
   constructor(reference, snapshot) {
diff --git a/node_modules/@react-native-firebase/database/lib/DatabaseOnDisconnect.js b/node_modules/@react-native-firebase/database/lib/DatabaseOnDisconnect.js
index 830edf3..e0c3aed 100644
--- a/node_modules/@react-native-firebase/database/lib/DatabaseOnDisconnect.js
+++ b/node_modules/@react-native-firebase/database/lib/DatabaseOnDisconnect.js
@@ -24,7 +24,7 @@ import {
   isUndefined,
   isValidPath,
   promiseWithOptionalCallback,
-} from '@react-native-firebase/app/lib/common';
+} from '@react-native-firebase/app/lib/common/index.js';
 
 export default class DatabaseOnDisconnect {
   constructor(reference) {
diff --git a/node_modules/@react-native-firebase/database/lib/DatabaseQuery.js b/node_modules/@react-native-firebase/database/lib/DatabaseQuery.js
index 3c95d85..67f0c87 100644
--- a/node_modules/@react-native-firebase/database/lib/DatabaseQuery.js
+++ b/node_modules/@react-native-firebase/database/lib/DatabaseQuery.js
@@ -26,7 +26,7 @@ import {
   pathIsEmpty,
   pathToUrlEncodedString,
   ReferenceBase,
-} from '@react-native-firebase/app/lib/common';
+} from '@react-native-firebase/app/lib/common/index.js';
 import DatabaseDataSnapshot from './DatabaseDataSnapshot';
 import DatabaseSyncTree from './DatabaseSyncTree';
 
diff --git a/node_modules/@react-native-firebase/database/lib/DatabaseQueryModifiers.js b/node_modules/@react-native-firebase/database/lib/DatabaseQueryModifiers.js
index 618ed1a..95ee766 100644
--- a/node_modules/@react-native-firebase/database/lib/DatabaseQueryModifiers.js
+++ b/node_modules/@react-native-firebase/database/lib/DatabaseQueryModifiers.js
@@ -15,7 +15,7 @@
  *
  */
 
-import { isNull, isNumber, isString } from '@react-native-firebase/app/lib/common';
+import { isNull, isNumber, isString } from '@react-native-firebase/app/lib/common/index.js';
 
 const CONSTANTS = {
   VIEW_FROM_LEFT: 'left',
diff --git a/node_modules/@react-native-firebase/database/lib/DatabaseReference.js b/node_modules/@react-native-firebase/database/lib/DatabaseReference.js
index 0ebe011..ede1117 100644
--- a/node_modules/@react-native-firebase/database/lib/DatabaseReference.js
+++ b/node_modules/@react-native-firebase/database/lib/DatabaseReference.js
@@ -28,7 +28,7 @@ import {
   pathChild,
   pathParent,
   promiseWithOptionalCallback,
-} from '@react-native-firebase/app/lib/common';
+} from '@react-native-firebase/app/lib/common/index.js';
 import DatabaseDataSnapshot from './DatabaseDataSnapshot';
 import DatabaseOnDisconnect from './DatabaseOnDisconnect';
 import DatabaseQuery, {
diff --git a/node_modules/@react-native-firebase/database/lib/DatabaseSyncTree.js b/node_modules/@react-native-firebase/database/lib/DatabaseSyncTree.js
index a0947c2..51fa31c 100644
--- a/node_modules/@react-native-firebase/database/lib/DatabaseSyncTree.js
+++ b/node_modules/@react-native-firebase/database/lib/DatabaseSyncTree.js
@@ -15,7 +15,7 @@
  *
  */
 
-import { isString } from '@react-native-firebase/app/lib/common';
+import { isString } from '@react-native-firebase/app/lib/common/index.js';
 import { getReactNativeModule } from '@react-native-firebase/app/lib/internal/nativeModule';
 import NativeError from '@react-native-firebase/app/lib/internal/NativeFirebaseError';
 import SharedEventEmitter from '@react-native-firebase/app/lib/internal/SharedEventEmitter';
diff --git a/node_modules/@react-native-firebase/database/lib/DatabaseTransaction.js b/node_modules/@react-native-firebase/database/lib/DatabaseTransaction.js
index 03b0b65..88c9872 100644
--- a/node_modules/@react-native-firebase/database/lib/DatabaseTransaction.js
+++ b/node_modules/@react-native-firebase/database/lib/DatabaseTransaction.js
@@ -16,7 +16,7 @@
  */
 
 import NativeError from '@react-native-firebase/app/lib/internal/NativeFirebaseError';
-import { isOther } from '@react-native-firebase/app/lib/common';
+import { isOther } from '@react-native-firebase/app/lib/common/index.js';
 
 let transactionId = 0;
 
diff --git a/node_modules/@react-native-firebase/database/lib/index.js b/node_modules/@react-native-firebase/database/lib/index.js
index 73a58d0..921ba37 100644
--- a/node_modules/@react-native-firebase/database/lib/index.js
+++ b/node_modules/@react-native-firebase/database/lib/index.js
@@ -15,7 +15,7 @@
  *
  */
 
-import { isAndroid, isBoolean, isNumber, isString } from '@react-native-firebase/app/lib/common';
+import { isAndroid, isBoolean, isNumber, isString } from '@react-native-firebase/app/lib/common/index.js';
 import {
   createModuleNamespace,
   FirebaseModule,
